docs/api.md

```diff
@@ -526,3 +527,66 @@ coll.use(new TitleCharTransformer("?"));
 There's currently no way to deal with adding tranformers to an already filled remote database; that would mean remote data migrations, and both Kinto and Kinto.js don't provide this feature just yet.
 
 **As a rule of thumb, you should only start using transformers on an empty remote collection.**
+
+## Id schemas
+
+By default, kinto.js uses UUID4 strings for record id's. If you want to work with an existing body of data, this may not be what you want.
+
+You can define a custom id schema on a collection by calling the `Collection#use()` method, which accepts a `Kinto.IdSchema`-derived object instance:
+
+```js
+import Kinto from "kinto";
+
+class IntegerIdSchema extends IdSchema {
+  constructor() {
+    super();
+    this._next = 0;
+  }
+  generate() {
+    return this._next++;
+  }
+  validate(id) {
+    return ((id == parseInt(id, 10)) && (id >= 0));
+  }
+}
+
+const kinto = new Kinto({remote: "https://my.server.tld/v1"});
+coll = kinto.collection("articles");
+coll.use(new IntegerIdSchema());
+```
+
+> #### Notes
+>
+> - The `generate` method should generate unique id's;
+> - The `validate` method should return a boolean, where `true` means valid.
+
+### Creating id schemas in ES5
+
+If your JavaScript environment doesn't suppport [ES6 classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) just yet, you can derive id schemas in an ES5 fashion using `Kinto.createIdSchema` static helper:
+
+```js
+var IntegerIdSchema = Kinto.createIdSchema({
+  constructor() {
+    this._next = 0;
+  }
+  generate() {
+    return this._next++;
+  }
+  validate(id) {
+    return ((id == parseInt(id, 10)) && (id >= 0));
+  }
+});
+
+coll.use(new IntegerIdSchema());
+```
+
+> #### Notes
+>
+> Notice you can define a `constructor` method, though you don't need (and must not) call `super()` within it, unlike with ES6 classes.
+
+### Limitations
+
+There's currently no way to deal with adding a custom id schema to an already filled remote database; that would mean remote data migrations, and both Kinto and Kinto.js don't provide this feature just yet.
```

Comment:
> There's currently no way to deal with adding a custom id schema to an already filled remote database

Isn't it what we are doing with Syncto? I think we should specify that there are no way to switch from UUID4 to custom id schema because if your remote database already respect the schema there are no limitation.